core/src/main/scala/org/apache/spark/internal/config/package.scala

```diff
@@ -563,4 +563,12 @@ package object config {
       .intConf
       .checkValue(v => v > 0, "The value should be a positive integer.")
       .createWithDefault(2000)
+
+  private[spark] val BUFFER_PAGE_SIZE =
```

Comment:
Small comment, while this is a valid change to make, it is a bit unrelated. Shall we do this in a different PR?